cmd/earthly/main.go

```diff
@@ -1176,6 +1176,86 @@ Set up a whole custom git repository for a server called example.com, using a si
 				},
 			},
 		},
+		{
+			Name: "preview",
+			Subcommands: []*cli.Command{
+				{
+					Name:        "project",
+					Description: "Create and manage Earthly projects",
+					UsageText:   "earthly project (ls|rm|create|member)",
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     "org",
+							EnvVars:  []string{"EARTHLY_ORG"},
+							Usage:    "The name of the organization to which the project belongs. Required when user is a member of multiple.",
+							Required: false,
+						},
+					},
+					Subcommands: []*cli.Command{
+						{
+							Name:        "ls",
+							Aliases:     []string{"list"},
+							Usage:       "List all projects that belong to the specified organization",
+							Description: "List all projects that belong to the specified organization",
+							UsageText:   "earthly project --org <organization-name> ls",
+							Action:      app.actionProjectList,
+						},
+						{
+							Name:        "rm",
+							Aliases:     []string{"remove", "delete"},
```

Comment:
I saw them for `ls`/`list` so I just tossed them in. I'm fine to remove these aliases if they're deemed superfluous. Also curious what @vladaionescu and @alexcb think.

Reply:
We could `remove` and `delete` as aliases to the other `rm` commands as well if we think they are convenient for the user

Reply:
Aliases seem fine, but we should add them everywhere for consistency. Maybe just remove them for this PR and add them in everywhere if we ever get feedback from a user? I have no strong opinion rather than being consistent.

Reply:
I think less is more in cases like these.

Maybe even for `ls` I would only keep one option, unless it's for backwards compatibility reasons, or for ergonomics reasons. Otherwise, it's just added noise.